commands/utils/quick-pick-image.ts

```diff
@@ -78,3 +81,51 @@ export async function quickPickImage(actionContext: IActionContext, includeAll?:
         return response;
     }
 }
+
+export async function quickPickImageName(actionContext: IActionContext, rootFolder: vscode.WorkspaceFolder, dockerFileItem: Item | undefined): Promise<string> {
+    let absFilePath: string = path.join(rootFolder.uri.fsPath, dockerFileItem.relativeFilePath);
+    let dockerFileKey = `ACR_buildTag_${absFilePath}`;
+    let prevImageName: string | undefined = ext.context.globalState.get(dockerFileKey);
+    let suggestedImageName: string;
+
+    if (!prevImageName) {
+        // Get imageName based on name of subfolder containing the Dockerfile, or else workspacefolder
+        suggestedImageName = path.basename(dockerFileItem.relativeFolderPath).toLowerCase();
+        if (suggestedImageName === '.') {
+            suggestedImageName = path.basename(rootFolder.uri.fsPath).toLowerCase().replace(/\s/g, '');
+        }
+
+        suggestedImageName += ":{{.Run.ID}}"
```

Comment:
Were you trying to fill in this value?

![image](https://user-images.githubusercontent.com/6913354/48234741-bdccf100-e36f-11e8-9184-6d149215c9b7.png)

Reply:
yes. This was a request from Steve Lasker. 
The repo name to be the folder name and the tag to be the run id by default, and we don't get a runID until we schedule the run, so this is the way with ACR to create the image with the run ID as the tag.

Reply:
hahaha. Yes, I see what you mean. I felt the same way the first time a saw it in one of the template yaml files.